src/Calypso-SystemTools-FullBrowser/ClySwitchToProjectsCommand.class.st

```diff
@@ -11,6 +11,12 @@ Class {
 	#category : #'Calypso-SystemTools-FullBrowser-Commands-Packages'
 }
 
+{ #category : #testing }
+ClySwitchToProjectsCommand class >> canBeExecutedInContext: aContext [
+
+	^ ClySystemEnvironment currentImage projectManager projectManagers notEmpty 
```

Comment:
Can you refactor it to have simple #isProjectsSupported?

Reply:
Does it have any value? All the project support has to be reviewed and redone in any case...